src/templates/docker-compose.yml.j2

```diff
@@ -137,11 +137,18 @@ services:
       - ./authelia/config:/config
     restart: unless-stopped
 
-{% if pinanas.applications is not none -%}
-  {% macro docker_compose() -%}
-    {% for apps in pinanas.applications %}
-      {% include "./docker-templates/" + apps + ".yml" -%}
-    {% endfor %}
-  {% endmacro %}
-  {{ docker_compose()|indent(2) }}
+{% if pinanas.applications is none -%}
+  {% for file_name in listdir(./docker-templates/) %}
+    {% filter indent(width=2) %}
+    {% include "./docker-templates/" + file_name + ".yml" -%}
```

Comment:
Should it not be only `{% include file_name %}`?
I expect the directory and the extension to be part of the `file_name` variable.
Side note, it is common to use as a variable name:
- `filename` for the name of a file
- `filepath` for the complete path (relative or absolute) of a file.

Reply:
Only way I find to resolve that is to list all "optionnal apps" in "default settings". Jinja seem not able to list name of files in a directory.